public/javascripts/ng-controllers.js

```diff
@@ -97,6 +97,7 @@ springInitiative.controller('mainController', function($scope, $http) {
       })
   };
 
+  // TODO: .success and .error are deprecated
```

Comment:
Can someone explain this - I've never heard this.  Also, if that is the case, what is the best way to fix it?